modules/github-oidc-role/versions.tf

```
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.0.0"
```

Use optional object members and require Terraform 1.3.0 to support that.
```suggestion
  required_version = ">= 1.3.0"
```